onnx/defs/tensor/defs.cc

```diff
@@ -769,18 +769,31 @@ ONNX_OPERATOR_SET_SCHEMA(
 static const char* Slice_ver13_doc = R"DOC(
 Produces a slice of the input tensor along multiple axes. Similar to numpy:
 https://docs.scipy.org/doc/numpy/reference/arrays.indexing.html
-Slices uses `starts`, `ends`, `axes` and `steps` inputs to specify the start and end
```

Comment:
I wonder if https://numpy.org/doc/stable/user/basics.indexing.html?highlight=slice#slicing-and-striding is a better link?

Reply:
A link to NumPy would make more sense for something preceded by "Similar to numpy".